common/src/main/java/com/wynntils/core/config/ConfigManager.java

```diff
@@ -169,13 +152,22 @@ public void loadConfigOptions(boolean resetIfNotFound, boolean initOverlayGroups
         }
 
         // Newly created group overlays need to be enabled
-        for (OverlayGroupHolder holder : OVERLAY_GROUP_FIELDS) {
-            holder.getParent().enableOverlays();
+        for (OverlayGroupHolder holder : Managers.Overlay.getOverlayGroups()) {
+            Managers.Overlay.enableOverlays(holder.getParent());
         }
     }
 
     private static List<ConfigHolder> getConfigHolderList() {
-        return Stream.concat(CONFIG_HOLDERS.stream(), OVERLAY_GROUP_CONFIG_HOLDERS.stream())
+        // FIXME: This breaks the concept of "manager holds all config holders at all times". Instead we get the group
```

Comment:
This is a FIXME since you indent to address this in a follow up? Or do you really accept this design but want to point out that it is not ideal? If so, you can keep the comment but remove the FIXME.

Reply:
I intend to fix this up shortly, but I removed the FIXME anyways.